inst/shiny/ui.R

```diff
@@ -36,15 +36,240 @@ shinyUI(
     #bootstrap theme
     theme = "bootstrap.min.css",
     #Upload Tab
+<<<<<<< HEAD
```

Comment:
Need to resolve all conflicts like this one and remove all conflict markers https://help.github.com/articles/resolving-a-merge-conflict-on-github/